OrbitLinuxTracing/PerfEvent.h

```diff
@@ -185,6 +185,58 @@ struct dynamically_sized_perf_event_stack_sample {
       : stack{dyn_size} {}
 };
 
+struct dynamically_sized_perf_event_call_chain_sample {
+  struct dynamically_sized_perf_event_call_chain {
+    uint64_t dyn_size;
```

Comment:
Isn't `dyn_size` for `PERF_SAMPLE_STACK_USER`? You should use `nr` like the manpage (maybe with a comment, but reusing another name is confusing).